client/components/jetpack/connection-health/index.tsx

```diff
@@ -42,27 +33,27 @@ export const JetpackConnectionHealthBanner = ( { siteId }: Props ) => {
 		return;
 	}
 
-	return (
-		<>
-			<TrackComponentView
-				eventName="calypso_jetpack_connection_health_issue_view"
-				eventProperties={ { type: 'default' } }
+	if ( jetpackConnectionHealth?.error === DNS_ERROR ) {
+		return (
+			<ErrorNotice
+				eventType="dns"
```

Comment:
Can we call this `errorType` instead of `eventType`? Also, we should probably pass the value of the constant for consistency, instead of coming up with some 'dns' shorthand version.

Reply:
FYI - I'm renaming this prop in https://github.com/Automattic/wp-calypso/pull/80292

Reply:
Thanks for clarifying that. I rebased this branch on the trunk and renamed properties as proposed.